gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Jul 23 15:06:18 CEST 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Please don't use `-all` distribution. Can we revert this file?

Reply:
This was already outdated